server/test/support/FakeReadOnlyQuestionService.java

```diff
@@ -0,0 +1,50 @@
+package support;
+
+import com.google.common.collect.ImmutableList;
+import services.question.ActiveAndDraftQuestions;
+import services.question.ReadOnlyQuestionService;
+import services.question.exceptions.QuestionNotFoundException;
+import services.question.types.EnumeratorQuestionDefinition;
+import services.question.types.QuestionDefinition;
+
+public class FakeReadOnlyQuestionService implements ReadOnlyQuestionService {
+
+  private final ImmutableList<QuestionDefinition> questions;
+
+  public FakeReadOnlyQuestionService(ImmutableList<QuestionDefinition> questions) {
+    this.questions = questions;
+  }
+
+  @Override
+  public ImmutableList<QuestionDefinition> getAllQuestions() {
+    return questions;
+  }
+
+  @Override
+  public ImmutableList<QuestionDefinition> getUpToDateQuestions() {
+    return questions;
+  }
+
+  @Override
+  public ImmutableList<EnumeratorQuestionDefinition> getAllEnumeratorQuestions() {
+    return null;
```

Comment:
I'm hesitant to implement them because they're currently unused and the semantics of these methods is implementation-dependent, given there are a couple different implementations of `ReadOnlyQuestionService`. I'd rather if someone uses these methods in test they decide what the correct behavior is.

Reply:
Switched to `throw new NotImplementedException` for the other two methods.